Fix sentence fragment in _get-sdk.md (#3531)
flutter/website: src/docs/get-started/install/_get-sdk.md

    $ {{unzip}} ~/Downloads/flutter_{{os}}_vX.X.X-{{site.sdk.channel}}{{file_ext}}
    ```
    

>     If you don't want to install a fixed version of the installation bundle, steps 1 and 2 can be replaced. 

Reviewer: ```suggestion
    If you don't want to install a fixed version of the installation bundle, 
    you can skip steps 1 and 2. 
    Instead, get the source code from the [Flutter repo](https://github.com/flutter/flutter) on GitHub,
    and change branches or tags as needed.
```

>     The Flutter SDK is free and open source, so you can get the source code from the [Flutter repo](https://github.com/flutter/flutter) on GitHub,

Reviewer: ```suggestion
```

>     and change branches or tags as needed.

Reviewer: ```suggestion

```
